src/index.ts

```diff
@@ -34,8 +36,20 @@ Array.prototype.sum = function (func = x => x, thisArg = this) {
   return thisArg.reduce<number>((acc, t: any) => acc + func(t), 0);
 };
 
-Array.prototype.average = function (func = x => x, thisArg = this) {
+Array.prototype.average = function (
+  func = x => x,
+  round = null,
+  thisArg = this
+) {
   if (thisArg.length <= 0) throw Error("Out of bounds");
+  if (round !== null) {
+    if (round <= 0) throw Error("Out of bounds");
```

Comment:
Couldn't `round == 0` be allowed? It could be taken as the same case as `round == null`.